### PR TITLE
Add 73 new sources

### PR DESCRIPTION
**Pull request by norric-funnel**
Another 73 new sources from Funnel, including 68 new organizations.

Thanks

### CODE REVIEW

File: organizations/powerinbox.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "id": "POWERINBOX",
+  "name": "PowerboxInbox",
```
Comment (pfrisella) [inline organizations/powerinbox.json]: s/PowerboxInbox/PowerInbox

File: sources/2_dehands.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
+  "id": "2DEHANDS",
```
Comment (pfrisella) [inline sources/2_dehands.json]: filename should be 2dehands

File: sources/powerinbox.json [hunk 3]
```diff
@@ -0,0 +1,10 @@
+{
+  "id": "POWERINBOX",
+  "name": "PowerboxInbox",
```
Comment (pfrisella) [inline sources/powerinbox.json]: PowerInbox ?

**Comment by fouxarn** [comment]
@googlebot I consent.

**Pull request merged**